tests/python_parser/test_syntax_error_handling.py

```diff
@@ -22,8 +24,17 @@ def parse_invalid_syntax(python_parse_file, python_parse_str, tmp_path, source,
     with pytest.raises(exc_cls) as e:
         python_parse_file(str(test_file))
 
-    print(str(e.exconly()))
-    assert message in str(e.exconly())
+    print(str( e.exconly()))
+    assert message in str( e.exconly())
+
+    exc = e.value
+    if start:
+        assert exc.lineno == start[0]
+        assert exc.offset == start[1]
+
+    if stop and sys.version_info >= (3, 10):
```

Comment:
Nice!

Reply:
Thanks. However adding actual test values will take me some time. Feel free to add some yourself if you want.